app/assets/stylesheets/global/african_poetics.scss

```diff
@@ -425,7 +423,89 @@ svg path.re {
   opacity: 0;
 }
 
-// African Poets and Poetry in the News
+// ------------
+// Contemporary African Poetry
+// ------------
+
+// Author pages
+
+.author_data {
+    margin-bottom: 1.8em;
+    margin-top: .5em;
+  h4 {
+    margin-top:-.15em;
```

Comment:
Space between property name and value. Same below on `.item_subtitle`'s `margin-top` property